src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -93,6 +96,16 @@ public ProcessResult updateCase(
 
             final CaseDetails existingCase = startEvent.getCaseDetails();
 
+            if (isExceptionAlreadyAttached(existingCase, exceptionRecord)) {
+                log.warn(
+                    "Exception already attached to case. "
```

Comment:
Maybe it would be worth mentioning that we're skipping update because all documents from this exception record are already in the case. That's to distinguish between this scenario and one where we check by `attachedToCaseReference` field.